src/language/typescript/common/data/serialized-fragment.ts

```diff
@@ -136,12 +139,19 @@ export const getSerializedOptionCallFragment = (
 					`pipe(
 							${a},
 							option.fromNullable,
-							option.map(${fn}),
+							option.chain(${fn}),
 						)`,
 					[serializedDependency('option', 'fp-ts'), serializedDependency('pipe', 'fp-ts/lib/pipeable')],
 					[],
 			  )
-			: serializedFragment(`some((${fn})(${a}))`, [serializedDependency('some', 'fp-ts/lib/Option')], []),
+			: serializedFragment(
+					`pipe(
```

Comment:
minor: is it really necessary to change direct function invocation to pipe?

Reply:
It's just a matter of readability; direct invocation turns out less readable even after applying prettier. Shall I switch it back to the direct call?
![image](https://user-images.githubusercontent.com/1782645/79320818-10222200-7f13-11ea-801d-adbc9167dcdf.png)

Reply:
let's skip